packages/refine/package-for-release.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@recoiljs/refine",
+  "publishConfig": {
```

Comment:
I don't think the other `package.json`'s had this.  Is it necessary or should we add it to the others?  Or is it needed here because it is a scoped package?

Reply:
It's because it's a scoped package. Scoped packages are private by default